### PR TITLE
[locale] fo: Fixed relativeTimes for m and M (#4609)

### PR DESCRIPTION
**Pull request by Sakarisson**
Fixed relativeTimes issues described in issue #4609.

### CODE REVIEW

**Comment by Sakarisson** [comment]
@ichernev
@ragnar123

**Comment by Sakarisson** [comment]
It appears that the original author is inactive on GitHub. Is there any fallback for this case?

**Comment by marwahaha** [comment]
@Sakarisson  - we don't have a formal process, but maybe these will work:
1) can you find other sources that confirm your change?
2) can you join on as a "locale author"? please add your github name to the file.

**Comment by Sakarisson** [comment]
@marwahaha - Thanks for the response.

1. What kind of sources are you thinking about? I linked to an online dictionary in my issue. I can probably also contact some source at the university to verify the translation.
2. Will do.

File: src/locale/fo.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! moment.js locale configuration
 //! locale : Faroese [fo]
-//! author : Ragnar Johannesen : https://github.com/ragnar123
+//! author : Ragnar Johannesen, Kristian Sakarisson : https://github.com/ragnar123, https://github.com/sakarisson
```
Comment (marwahaha) [inline src/locale/fo.js]: Ah, I would prefer your name on a new line, like
https://github.com/moment/moment/blob/develop/src/locale/ro.js

**Pull request merged**